expr/types.go

```diff
@@ -144,6 +144,17 @@ type ArgStringsOrInts struct {
 func (a ArgStringsOrInts) Key() string    { return a.key }
 func (a ArgStringsOrInts) Optional() bool { return a.opt }
 
+// mixed string or int
+type ArgStringOrInt struct {
```

Comment:
Wouldn't this need to be added to the parsing section of expr/expr.go?

Reply:
You mean consumeBasicArg() in expr/expr.go? Yes it should, I think I missed this part.

this function verifies the basic args, so if I didn't add the new type into this, it should fail. However I still got the correct results. Does this mean the query was routed to Graphite?

Reply:
ooh, probably. They recently added logic to proxy to graphite on more error types.

Reply:
cool I have added the missing part in expr.go